### ISSUE TITLE
Button missing a text alternative

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_directories#261]
@Mjeffbrown reports (from Siteimprove)

Button missing a text alternative - The map pins on directories don't seem to have alt text.

**Comment by ekes** [comment]
> Button missing a text alternative

Which one? The only button I think of is the search 'Apply' which is a standard input button with 'Apply' as its value.

>  The map pins on directories don't seem to have alt text

Does that even make sense to have alt text. The whole map is an image?

**Comment by msayoung** [comment]
Well...

The pins on the map are `<img>` s with the `role="button"` . This makes sense as they can be pressed to give us the link to the directory entry. 

eg https://demo.localgovdrupal.org/localgov-drupal-collaborators

```
<img src="https://demo.localgovdrupal.org/modules/contrib/leaflet/js/leaflet/dist/images/marker-icon.png" class="leaflet-marker-icon leaflet-zoom-animated leaflet-interactive" title="Annertech" alt="Annertech" tabindex="0" role="button" style="margin-left: -12px; margin-top: -41px; width: 25px; height: 41px; transform: translate3d(211px, 167px, 0px); z-index: 167; outline: none;">

```
The image does have alt text, but it seems that SiteImprove isn't picking that up at the text alternative for the button. We'll need to do some testing to see what impact this has on assistive technologies (eg screen readers) . 


However, I'd say a bigger problem is the focus order when you click the button - you have to tab through all the markers before you get to the popup created by clicking. I'll test that too and create another ticket if needed.

**Comment by markconroy** [comment]
I've tested this on Voiceover on Mac and the alt text for the icons is being read out. I guess we should test it with other screen readers as well.

HTML_Code_Sniffer reports:
> This element has role of "button" but does not have a name available to an accessibility API. Valid names are: element content, aria-label undefined, aria-labelledby undefined.

Axe Dev Tools reports no issue.

Wave  reports no issue.
